framework/jobs/src/org/apache/cloudstack/framework/jobs/impl/AsyncJobManagerImpl.java

```diff
@@ -762,12 +768,34 @@ protected void runInContext() {
             protected void reallyRun() {
                 try {
                     List<SyncQueueItemVO> l = _queueMgr.dequeueFromAny(getMsid(), MAX_ONETIME_SCHEDULE_SIZE);
+                    boolean isPurged = false;
```

Comment:
It will run again in the next 2 seconds. I dont remember why its out of for loop. may be there were some issues with the dequeue. I move it in and see.